**Ishak Yusuf**  
*Computational Biologist*  

**Contact Information:**  
Email: ishakyusuf18@outlook.com  
LinkedIn: [isky20](https://www.linkedin.com/in/isky20/)  
ORCID: [0000-0003-1635-1350](https://orcid.org/0000-0003-1635-1350)  

---

**ABOUT**

I am an experienced Computational Biologist with over five years of expertise in analyzing diverse biological data. I hold a Master of Science degree in Bioinformatics and Molecular Genetics from the University of Leicester and a Bachelor's degree in Biotechnology from UCSI University. Currently, I am a Research Fellow at the Istituto di Tecnologie Biomediche. My work focuses on evaluating bioinformatics tools for omics data analysis, particularly in proteomics and transcriptomics. Through my research, I aim to unravel complex biological systems and advance our understanding of crucial cellular processes, significantly contributing to scientific progress and innovation in bioinformatics and biomedical research.

---

**EXPERIENCE**

*Research Fellow (Computational Biologist)*  
Istituto di Tecnologie Biomediche, CNR  
Milan, Italy  
**Nov 2022 – Present**  
- Processed omics data, particularly proteomics and transcriptomics, across various contexts, including clinical and plant studies.
- Analyzed protein expression, modifications, and interactions using liquid chromatography and mass spectrometry.
- Utilized RNA sequencing for studying gene expression changes.
- Employed visualization methods like bar plots, gochord, ggtree, and PPI network models.
- Investigated protein-protein interactions (PPI) and miRNA targets for gene regulation analysis.
- Used Gene Ontology (GO) enrichment to categorize genes into functional groups.

*Bioinformatics Scientist*  
Genoks Genetics Centre  
Ankara, Turkey  
**Jun 2021 – Oct 2022**  
- Analyzed multi-omics data with skills in R, bash, and Python for data manipulation and visualization.
- Optimized pipelines on HPC systems for efficient analysis of large sample sets.
- Worked on Whole Genome Sequencing (WGS) and Whole Exome Sequencing (WES) projects, including variant annotation.
- Communicated results through detailed reports and visualizations.

---

**EDUCATION**

*Master of Science in Molecular Genetics and Bioinformatics*  
University of Leicester  
Leicester, England  
**Sep 2019 – Oct 2020**  
- **Grade:** Merit  
- **MSc Project:** Optimising an NLP algorithm to extract table data from GWAS publications.

*Bachelor of Science in Biotechnology*  
UCSI University  
Kuala Lumpur, Malaysia  
**Oct 2014 – Aug 2018**  
- **GPA:** 2.81  
- **Research Project:** Determination of sub-cytotoxic dose range of fungal immunomodulatory protein for potential neurodegenerative treatment.

---

**PROJECTS**

*Cardio Regeneration by miRNA Therapy*  
- Analyzed transcriptomics and proteomics data for evaluating the effect of miRNAs on cardiomyocyte proliferation.
- Used databases like multiMiR, TargetScan, and miRTarBase for miRNA target identification.
- Employed Cytoscape for PPI network construction and analysis.

*Transcriptomics in Multiple Sclerosis (MS)*  
- Analyzed gene expression data from peripheral and induced T cells in MS patients.
- Used tools like FastQC, STAR, and MultiQC for quality assessment and alignment checks.
- Compared FoxP3-targeted genes with differentially expressed genes to understand molecular effects of treatments on T cell gene expression.

*Prote/Transcriptomics Analysis in Duchenne Muscular Dystrophy (DMD)*  
- Investigated the impact of Givinostat on DMD using mouse models.
- Analyzed transcriptomics and proteomics data to characterize miRNAs with a key role.

*Transcriptomics in Arabidopsis thaliana (plant)*  
- Investigated the transcriptome in the context of retrograde signaling and Unfolded Protein Response (UPR).
- Visualized differential gene expression using ggplot and built PPI networks.

*Proteomic Analysis in Neuroblastoma*  
- Analyzed proteomic data from extracellular vesicles (EVs) released by neuroblastoma cells treated with OLIGP GM1 peptides.
- Conducted enrichment analysis and constructed PPI networks for insights into molecular effects.

*Proteomic Analysis in Prostate Cancer*  
- Investigated proteins in urine samples from healthy controls and prostate cancer patients.
- Conducted enrichment analysis and visualized significant terms using custom R scripts and bioinformatics tools.

---

**SKILLS**

- Critical analysis of research publications and experimental design.
- Proficient in Linux/Unix environments, R, and Python for bioinformatics and statistical analysis.
- Skilled in debugging Python scripts.
- Experienced in Snakemake for managing complex workflows.
- Proficient with bioinformatics tools: FastQC, Trimmomatic, STAR, DESeq2, GATK, BLAST, Bowtie2, VCFtools, and Docker.
- Strong knowledge of biostatistics and machine learning applications in healthcare.
- Experience with WGS, WES, and GWAS methodologies, including experimental execution and data analysis.

---

**PUBLICATIONS**

- Review: The Role of Learning Approaches in Transcriptomics/Proteomics Data (in process)

---

**REFERENCES**

- Available upon request.
